azure-pipelines-deploy-template.yml

```
@@ -346,7 +346,32 @@ jobs:
               SpecifySlotOrASE: true
               ResourceGroupName: '$(resourceGroupName)'
               Slot: staging
+          
+          - task: AzurePowerShell@4
+            displayName: 'GET /check in $(appServiceName)-staging'
+            inputs:
+              azureSubscription: '${{parameters.subscriptionName}}'
+              ScriptType: InlineScript
+              azurePowerShellVersion: LatestVersion
+              Inline: |
+                Param(
+                  [string] $appServiceName = "$(appServiceName)"
+                )
+                $healthCheckUrl = [string]::Format(“https://{0}-staging.azurewebsites.net/check”, $appServiceName)
```

These are smart quotes, I don't think they would work, would they?

```suggestion
                $healthCheckUrl = [string]::Format("https://{0}-staging.azurewebsites.net/check", $appServiceName)
```